forest/constants.py

```diff
@@ -20,6 +20,9 @@ class Frequency(Enum):
     HOURLY = "hourly"
     DAILY = "daily"
     BOTH = "both"
+    THREE_HOURLY = "three_hourly"
+    SIX_HOURLY = "six_hourly"
+    TWELVE_HOURLY = "twelve_hourly"
```

Comment:
What does "both" mean in the context of these additional frequencies? Should that be something like `HOURLY_AND_DAILY` now?
Also, would it now make sense to assign `1`, `3`, `6`, `12`, and `24` as the enum values? Then you could simply do: `window = frequency * 60 * 60`